lightning/src/ln/channelmanager.rs

```diff
@@ -1876,7 +1876,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		};
 
 		for htlc_source in failed_htlcs.drain(..) {
-			self.fail_htlc_backwards_internal(self.channel_state.lock().unwrap(), htlc_source.0, &htlc_source.1, HTLCFailReason::Reason { failure_code: 0x4000 | 8, data: Vec::new() });
+			self.fail_htlc_backwards_internal(self.channel_state.lock().unwrap(), htlc_source.0, &htlc_source.1, HTLCFailReason::Reason { failure_code: 0x4000 | 8, data: Vec::new() }, HTLCDestination::NextHopChannel { node_id: *counterparty_node_id, channel_id: *channel_id });
```

Comment:
Could you use a separate variable for the destination to help keep the line lengths sane? I realize we haven't done a great job at this here, but might as well not add to the problem lol. Likewise throughout.